android/app/src/main/java/com/audiobookshelf/app/data/CollapsedSeries.kt

```diff
@@ -0,0 +1,36 @@
+package com.audiobookshelf.app.data
+
+import android.content.Context
+import android.os.Bundle
+import android.support.v4.media.MediaDescriptionCompat
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+class CollapsedSeries(
+  id:String,
+  var libraryId:String?,
+  var name:String,
+  //var nameIgnorePrefix:String,
+  var sequence:String?,
+  var libraryItemIds:MutableList<String>
+) : LibraryItemWrapper(id) {
+  @get:JsonIgnore
+  val title get() = name
+  @get:JsonIgnore
+  val numBooks get() = libraryItemIds.size
+
+  @JsonIgnore
+  override fun getMediaDescription(progress:MediaProgressWrapper?, ctx: Context): MediaDescriptionCompat {
```

Comment:
I can't tell if this function is being used anywhere. I was looking because I was checking if `__SERIES__` was intentionally different than the mediaId used in the LibraryItem.getMediaDescription